assets/xml/sequences/Sequences.xml

```diff
@@ -0,0 +1,112 @@
+    <Sequence Name="043_Anticipating_Treasure"/>
+    <Sequence Name="044_Forest_Temple"/>
+    <Sequence Name="045_Hyrule_Castle_Courtyard"/>
+    <Sequence Name="046_Ganondorf_Theme_Organ"/>
+    <Sequence Name="047_Lon_Lon_Ranch"/>
+    <Sequence Name="048_Goron_City"/>
+    <Sequence Name="049_Hyrule_Field_Morning"/>
+    <Sequence Name="050_Got_Spiritual_Stone"/>
+    <Sequence Name="051_Bolero_of_Fire"/>
+    <Sequence Name="052_Minuet_of_Forest"/>
+    <Sequence Name="053_Serenade_of_Water"/>
+    <Sequence Name="054_Requiem_of_Spirit"/>
+    <Sequence Name="055_Nocturne_of_Shadow"/>
+    <Sequence Name="056_Mini_Boss_Battle"/>
+    <Sequence Name="057_Got_Item"/>
+    <Sequence Name="058_Temple_of_Time"/>
+    <Sequence Name="059_Rescued_Epona"/>
+    <Sequence Name="060_Kokiri_Forest"/>
+    <Sequence Name="061_Got_Fairy_Ocarina"/>
+    <Sequence Name="062_Lost_Woods"/>
+    <Sequence Name="063_Spirit_Temple"/>
+    <Sequence Name="064_Horse_Minigame"/>
+    <Sequence Name="065_Horse_Minigame_Goal"/>
+    <Sequence Name="066_Ingos_Theme"/>
+    <Sequence Name="067_Got_Medallion"/>
+    <Sequence Name="068_Sarias_Song"/>
+    <Sequence Name="069_Eponas_Song"/>
+    <Sequence Name="070_Zeldas_Lullaby"/>
+    <Sequence Name="071_Suns_Song"/>
+    <Sequence Name="072_Song_of_Time"/>
+    <Sequence Name="073_Song_of_Storms"/>
+    <Sequence Name="074_Navi_in_Flight"/>
+    <Sequence Name="075_Deku_Tree_Theme"/>
+    <Sequence Name="076_Windmill"/>
+    <Sequence Name="077_Legends_of_Hyrule"/>
+    <Sequence Name="078_Game_Shops"/>
+    <Sequence Name="079_Sheiks_Theme"/>
+    <Sequence Name="080_Zoras_Domain"/>
+    <Sequence Name="081_Zelda_Sting"/>
+    <Sequence Name="082_Zeldas_Theme_Ending"/>
+    <Sequence Name="083_Got_Master_Sword"/>
+    <Sequence Name="084_Ganondorfs_Theme"/>
+    <Sequence Name="085_Shop"/>
+    <Sequence Name="086_Chamber_of_Sages"/>
+    <Sequence Name="087_File_Select" Extract="false"/>
+    <Sequence Name="088_Ice_Cavern"/>
+    <Sequence Name="089_Opening_the_Door_of_Time"/>
+    <Sequence Name="090_Kaepora_Gaeboras_Theme"/>
+    <Sequence Name="091_Shadow_Temple"/>
+    <Sequence Name="092_Water_Temple"/>
+    <Sequence Name="093_The_Sages_Bridge"/>
+    <Sequence Name="094_Seal_of_the_Six_Sages"/>
+    <Sequence Name="095_Gerudo_Valley"/>
+    <Sequence Name="096_Potion_Shop"/>
+    <Sequence Name="097_Koume_and_Kotakes_Theme"/>
+    <Sequence Name="098_Escape_from_Ganons_Castle"/>
+    <Sequence Name="099_Ganondorfs_Castle"/>
+    <Sequence Name="100_Ganondorf_Battle"/>
+    <Sequence Name="101_Ganon_Battle"/>
+    <Sequence Name="102_Ending_Credits_1"/>
+    <Sequence Name="103_Ending_Credits_2"/>
+    <Sequence Name="104_Ending_Credits_3"/>
+    <Sequence Name="105_Ending_Credits_4"/>
+    <Sequence Name="106_Ending_Credits_5"/>
+    <Sequence Name="107_Dodongo_and_Volvagia_Battle"/>
+    <Sequence Name="108_Timed_Minigame"/>
+    <Sequence Name="109_Unknown" Extract="false"/>
```

Comment:
We know that this sequence is cutscene effects now right?

Reply:
Whoops, you're right, I'll fix that here.